.gitattributes

```diff
@@ -5,3 +5,4 @@ linking_flags.sh linguist-generated
 /dune-project    linguist-generated
 /sphinx/conf.py  linguist-generated
 /package.json    linguist-generated
+/**/version.mlt  linguist-generated
```

Comment:
@emilienlemaire Could you move these changes to a separate PR? (This kind of mismatch should not happen — caused by some hasty change in `drom`).

Reply:
Just this one or all the changes from all `version.mlt` as well?

Reply:
Ideally this one and the changes to `version.mlt`s.